sdk/src/main/java/io/dapr/internal/opencensus/BigendianEncoding.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.internal.opencensus;
+
+import java.util.Arrays;
+
+/**
+ * Code originally from https://github.com/census-instrumentation/opencensus-java/blob/
+ * 446e9bde9b1f6c0317e3f310644997e5d6d5eab2/api/src/main/java/io/opencensus/trace/BigendianEncoding.java
```

Comment:
Most of the files in this package are from opencensus. I copy and pasted the classes I needed, removed the methods that were unused and did some formatting. I also made almost all method to be package accessible. As per @yaron2, we should refer to the origin of the code but still use our license.